app/views/users/show.html.erb

```diff
@@ -1 +1,22 @@
-<h1>User details page</h1>
\ No newline at end of file
+<div class="user-details-page">
+<div class="row">
+  <div class="user-header">
+     <%= render partial: "shared/user_card", locals: {u: @user}%>
+  </div>
+
+  <div class="user-bio">
+     <h2>User bio</h2>
+    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+    <p>It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing</p>
+  </div>
+   
+   <%@posts.each do |p|%>
+      <%= render partial: "shared/post_card", locals: {p:p}%>
+   <%end%>
```

Comment:
- [ ] Kindly make sure to use methods that you have created in the models, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.) requirement. You can use the `three_most_recent_posts` method here to show the most recent posts instead of all of them.

- [ ] Kindly make sure to show posts that belong to the selected user and not all of the posts.